src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -75,7 +75,7 @@ private String getDefendantResponseIssuedEmailTemplate(Claim claim) {
         Response response = claim.getResponse().orElseThrow(() -> new IllegalStateException("Response expected"));
 
         if (isFullDefenceAndNoMediation(response)) {
```

Comment:
Do we need this condition? As the condition is contradicting with the return value. 

I am assuming we need to remove this condition and the below condition (line 81 onwards) will choose the correct template based on party type. Please clarify with business.

Reply:
Confirming this with business. 👍

Reply:
This template should be used for the journey:
I reject the claim -> I dispute all the claim -> no to mediation:
https://www.notifications.service.gov.uk/services/37ba0bfe-68ed-4e8e-9e19-f3a4cc331d80/templates/613ca230-ab62-4370-bacd-ecc65b92132a 

All other defendant response emails should be the same. However in the system there exists 2 email templates that have the exact same content. These templates are for defendant response individual/sole trader and company/organisation. Should I keep these 2 templates in the code or delete one? Will their email content be changed in the future- is it worth keeping the logic? Cheers

Reply:
If there is un-used template/code please delete that.

Reply:
Okay I'll remove unnecessary template and logic.

Yup that template will no longer be removed as it is needed for that journey 👍

Reply:
i did approve but now im a bit confused.com - Lauren you say `613ca230-ab62-4370-bacd-ecc65b92132a` should be used (I reject the claim -> I dispute all the claim -> no to mediation) but it's never used now, as you changed the code: `getDefendantResponseWithNoMediationIssued()`)... is this right?

Reply:
Yes there has been a change from business that's requiring me to bring back the template I was initially told to remove: `613ca230-ab62-4370-bacd-ecc65b92132a`.

I'm working on this currently but it hasn't been pushed, probably contributing to your confusion haha